Docs Changes for DevDot Migration (#1036)
hashicorp/terraform-plugin-sdk: website/docs/plugin/sdkv2/testing/index.mdx

Plugins. Tests are a vital part of the Terraform ecosystem, verifying we can
deliver on our mission to safely and predictably create, change, and improve
infrastructure. Documentation for Terraform tests are broken into categories
briefly described on this page. Each category has more detailed information on a separate page within the documentation.

> -> **Note:** Recent versions of Terraform CLI also support [developer overrides in the CLI configuration](/cli/config/config-file#development-overrides-for-provider-developers), which can be useful for manually testing providers. The acceptance testing framework uses real Terraform CLI executions, so developer overrides are only recommended as a last resort option for missing functionality.

Reviewer: ```suggestion
-> **Note:** Recent versions of Terraform CLI also support [developer overrides in the CLI configuration](/cli/config/config-file#development-overrides-for-provider-developers), which can be useful for manually testing providers. The acceptance testing framework uses real Terraform CLI executions. We only recommend developer overrides as a last resort option for missing functionality.
```

Suggest changing "are only recommended" to "We only recommend" to avoid passive voice.
